connect/basic-auth-extension/src/test/java/org/apache/kafka/connect/rest/basic/auth/extension/JaasBasicAuthFilterTest.java

```diff
@@ -53,110 +55,130 @@
     public void testSuccess() throws IOException {
         File credentialFile = setupPropertyLoginFile(true);
         JaasBasicAuthFilter jaasBasicAuthFilter = setupJaasFilter("KafkaConnect", credentialFile.getPath());
-        ContainerRequestContext requestContext = setMock("Basic", "user", "password", false);
+        ContainerRequestContext requestContext = setMock("Basic", "user", "password");
         jaasBasicAuthFilter.filter(requestContext);
```

Comment:
Don't we need any `verify` calls here?

Reply:
Yes, firstly I think it's good enough to verify `jaasBasicAuthFilter.filter(requestContext)` pass without any exception, after some consideration I think it's better to call `verify` for a double-check.